react.json

```diff
@@ -31,31 +31,32 @@
     "no-empty": 0,
     "no-restricted-syntax": 0,
     "no-await-in-loop": 0,
+    "no-param-reassign": 0,
+    "no-confusing-arrow": 0,
+    "no-nested-ternary": 0,
     "camelcase": 0,
     "arrow-parens": [
       "error",
       "as-needed"
     ],
-    "no-confusing-arrow": [
-      "error",
-      {
-        "allowParens": true
-      }
-    ],
     "import/extensions": 0,
     "import/no-unresolved": 0,
     "react/jsx-filename-extension": 0,
     "react/require-default-props": 0,
     "react/sort-comp": 0,
     "react/prop-types": 0,
     "react/jsx-closing-bracket-location": 0,
+    "react/forbid-prop-types": 0,
     "jsx-a11y/anchor-is-valid": 0,
+    "jsx-a11y/click-events-have-key-events": 0,
```

Comment:
i'll create an a11y audit task to address these issues (and the existing a11y issues that are `eslint-disabled`)

Reply:
created